rust/rope/src/delta.rs

```diff
@@ -101,7 +101,7 @@ impl<N: NodeInfo> Delta<N> {
                     end < self.base_len
                 } else if let DeltaElement::Copy(b1, e1) = self.els[1] {
                     // Deletion in middle
-                    end < b1 && e1 == self.base_len
+                   self.els.len() == 2 && end < b1 && e1 == self.base_len
```

Comment:
Just some general observation: wouldn't it be more expressive to use `elements` instead of `els`?

Reply:
I agree, and if we really want we could rename it to 'els' on the wire, if that's a concern? @raphlinus?